unlock-app/src/components/creator/CreatorLock.js

```diff
@@ -14,7 +14,7 @@ export function CreatorLock({ lock, status = 'deployed' }) {
   let lockComponentStatusBlock
 
   if (status === 'deployed') { // the transaction was mined and confirmed at least 12 times
-    lockComponentStatusBlock = <LockIconBar lock={lock} />
+    lockComponentStatusBlock = <LockIconBar lock={lock} className={'lock-icons'} />
```

Comment:
hum... I think we should not use `className` anymore with styled-components. (you'll soon hate me for being so picky).

The approach is to use   `${LockRow}:hover &` into the `IconBar` element. (Check what I did in `Button.js`)

Reply:
This approach doesn't work here, because of the way the components fit together, but I'm working on it ...